app/matplotlib.ipynb

```diff
@@ -86,23 +91,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.title(\"Random colors and sizes\")\n",
+    "plt.colorbar()\n",
```

Comment:
The task requires setting a label for the colorbar, which is currently missing. You can add it using `plt.colorbar(label='Your Label')`.